introduction.md

```diff
@@ -0,0 +1 @@
+this is an introduction, it is not a good introduction, but it is an introduction...
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.